index.html

```diff
@@ -342,6 +422,11 @@ <h2 id="tagline">Get Started</h2>
         </div>
         <input type="submit" value="Start collaboration" class="primary-btn" />
       </form>
+      <div class="image-geometry_7">
+        <a href="#"
+          ><img src="./assets/img/image-geometry_7.png" alt="logo_desktop"
+        /></a>
+      </div>
```

Comment:
- [ ] [OPTIONAL] Once again I'd like to apologize as the previous reviewer forgot to mention this but it seems like you forgot to add the bottom image in the "Get started" section as specified in the [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=t45uZqimwMP5BSMR-0)
    - Figma design:
      ![firefox_SaWm8KGpLf](https://user-images.githubusercontent.com/104968236/229143052-2f52ae20-ef69-451b-97c9-423bfb9e950d.png)
    - Your design:
      ![Inkedfirefox_e0mhYLCi7L](https://user-images.githubusercontent.com/104968236/229143628-0cae6f58-ed63-4315-8215-979f086527c7.jpg)
92a7b-4983-4a12-b8c5-4c6469e8018f.png)